src/common/components/home/FeaturedBuyNowListingsSection.tsx

```diff
@@ -0,0 +1,106 @@
+import { VFC } from 'react';
+import { NFTCard } from 'pages/profiles/[publicKey]/nfts';
+import { gql, useQuery } from '@apollo/client';
+import { HomeSection, HomeSectionCarousel } from 'pages/home-v2-wip';
+
+// TODO replace hardcoded list with gql query result
+const featuredListings: {address: string, marketplace: string}[] = [
+  {address: '9CZmL7zc87Qc4d8svJdVHjmmd5V9TVhGAqvziv7ibV1K', marketplace: 'junglecats'},
+  {address: '9CZmL7zc87Qc4d8svJdVHjmmd5V9TVhGAqvziv7ibV1K', marketplace: 'junglecats'},
+  {address: '9CZmL7zc87Qc4d8svJdVHjmmd5V9TVhGAqvziv7ibV1K', marketplace: 'junglecats'},
+  {address: '9CZmL7zc87Qc4d8svJdVHjmmd5V9TVhGAqvziv7ibV1K', marketplace: 'junglecats'},
+  {address: '9CZmL7zc87Qc4d8svJdVHjmmd5V9TVhGAqvziv7ibV1K', marketplace: 'junglecats'},
+  {address: '9CZmL7zc87Qc4d8svJdVHjmmd5V9TVhGAqvziv7ibV1K', marketplace: 'junglecats'},
+  {address: '9CZmL7zc87Qc4d8svJdVHjmmd5V9TVhGAqvziv7ibV1K', marketplace: 'junglecats'},
+  {address: '9CZmL7zc87Qc4d8svJdVHjmmd5V9TVhGAqvziv7ibV1K', marketplace: 'junglecats'},
+  {address: '9CZmL7zc87Qc4d8svJdVHjmmd5V9TVhGAqvziv7ibV1K', marketplace: 'junglecats'},
+];
+
+
+const FeaturedBuyNowListingsSection: VFC = () => {
+  return (
+    <HomeSection>
+      <HomeSection.Header>
+        <HomeSection.Title>What&apos;s Hot</HomeSection.Title>
+        <HomeSection.HeaderAction external href="https://google.com">
+          Discover All
+        </HomeSection.HeaderAction>
+      </HomeSection.Header>
+      <HomeSection.Body>
+        <HomeSectionCarousel rows={2} cols={3}>
+          {featuredListings.map((s) => (
+            <HomeSectionCarousel.Item key={s.address}>
+              <div className="p-2">
+                <NFTCardDataWrapper address={s.address} marketplace={s.marketplace}/>
+              </div>
+            </HomeSectionCarousel.Item>
+          ))}
+        </HomeSectionCarousel>
+      </HomeSection.Body>
+    </HomeSection>
+  );
+};
+
+
+const GET_NFT_DATA = gql`
```

Comment:
Is there a reason this is in the file and not a .graphql file with graphql codegen

Reply:
Just because it's placeholder stuff. Mayhap I should just delete it from the PR

Reply:
just removed from `home-v2-wip` so no danger of showing if someone finds the wip homepage